src/models/Org.model.ts

```diff
@@ -1,7 +1,7 @@
 import { prop } from '@typegoose/typegoose';
 
 export enum MemberAccessType {
-  INVITE_ONLY = 'invite_only',
+  INVITE_ONLY = 'inviteOnly',
```

Comment:
I think camelCase is fine for identifiers in source code, but looks really odd to use on an enum value or a DB field value.